phase2/outline
# Feature/breadcrumbs

Improves #42.

## Testing Notes
1. Open Storybook
2. Go to Navigation at the bottom of the left side
3. Choose Breadcrumbs
4. Observe that there are links
5. Hover over the first one
6. Observe that you see the color change to green with an underline
7. Observe that the last link has no `/` always has an underline
8. Observe that as you hover down the line the `/` remain blue.

## Example

<img width="1443" alt="Screen Shot 2021-08-13 at 3 38 23 PM" src="https://user-images.githubusercontent.com/216272/129410380-c9bb4f5f-6007-4ed5-bd68-d80ba1777de7.png">


## Notes

The markup is from [Drupal.org](https://www.drupal.org/association). I added the last and active classes to make it possible to change the behavior of those last two links. 

## Possible additional features

It would be great to have some variants of different types of dividers.

<a href="https://gitpod.io/#https://github.com/phase2/outline/pull/96"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
